### PR TITLE
Update array sintax from array() to []

### PR DESCRIPTION
**Pull request by mitogh**

### CODE REVIEW

**Review by lucatume — COMMENTED** [review]
This is really great work, rebase on `release/B20.11`?

File: src/Tribe/Main.php [hunk 1]
```diff
@@ -2085,88 +2136,88 @@ protected function generatePostTypeLabels() {
...
+					7  => sprintf( esc_html__( '%s saved.', 'the-events-calendar' ), $this->singular_organizer_label ),
+					8  => sprintf( esc_html__( '%s submitted.', 'the-events-calendar' ), $this->singular_organizer_label ),
+					9  => sprintf(
+							esc_html__( '%1$s scheduled for: %2$s.', 'the-events-calendar' ),
+							$this->singular_organizer_label,
+							// translators: Publish box date format, see http://php.net/date
+							'<strong>' . date_i18n( esc_html__( 'M j, Y @ G:i', 'the-events-calendar' ), strtotime( $post->post_date ) ) . '</strong>'
+					),
+					10 => sprintf( esc_html__( '%s draft updated.', 'the-events-calendar' ), $this->singular_organizer_label ),
+			];
```
Comment (paulmskim) [inline src/Tribe/Main.php:2220]: spacing

File: src/Tribe/Main.php [hunk 2]
```diff
@@ -2274,11 +2325,11 @@ public function cleanupPostOrganizers( $postId ) {
...
-			$the_query = new WP_Query( array(
-				'meta_key'   => $key,
-				'meta_value' => $postId,
-				'post_type'  => self::POSTTYPE,
-			) );
+			$the_query = new WP_Query( [
+					'meta_key'   => $key,
+					'meta_value' => $postId,
+					'post_type'  => self::POSTTYPE,
+			] );
```
Comment (paulmskim) [inline src/Tribe/Main.php:2332]: spacing

File: src/Tribe/Main.php [hunk 3]
```diff
@@ -2843,15 +2894,15 @@ public function googleCalendarLink( $post = null ) {
...
-			);
+			$params = [
+					'action'   => 'TEMPLATE',
+					'text'     => urlencode( strip_tags( $post->post_title ) ),
+					'dates'    => $dates,
+					'details'  => urlencode( $event_details ),
+					'location' => urlencode( $location ),
+					'trp'      => 'false',
+					'sprop'    => 'website:' . home_url(),
+			];
```
Comment (paulmskim) [inline src/Tribe/Main.php:2905]: spacing

File: src/Tribe/Main.php [hunk 4]
```diff
@@ -3317,10 +3369,10 @@ public function save_venue_data( $postID = null, $post = null ) {
...
 			 * external components like Community Plugin.
 			 */
-			$data = wp_parse_args( (array) $data, array(
-				'ShowMap' => 'false',
-				'ShowMapLink' => 'false',
-			) );
+			$data = wp_parse_args( (array) $data, [
+					'ShowMap'     => 'false',
+					'ShowMapLink' => 'false',
+			] );
```
Comment (paulmskim) [inline src/Tribe/Main.php:3375]: spacing

File: src/Tribe/Main.php [hunk 5]
```diff
@@ -3332,16 +3384,16 @@ public function save_venue_data( $postID = null, $post = null ) {
...
+		public function get_venue_info( $p = null, $args = [] ) {
+			$defaults = [
+					'post_type'            => Tribe__Events__Venue::POSTTYPE,
+					'nopaging'             => 1,
+					'post_status'          => 'publish',
+					'ignore_sticky_posts ' => 1,
+					'orderby'              => 'title',
+					'order'                => 'ASC',
+					'p'                    => $p,
+			];
```
Comment (paulmskim) [inline src/Tribe/Main.php:3396]: here and this entire file has weird indentation

File: src/views/modules/meta/details.php [hunk 6]
```diff
@@ -137,16 +137,17 @@
...
+				[
+						'before'       => '',
+						'sep'          => ', ',
+						'after'        => '',
+						'label'        => null, // An appropriate plural/singular label will be provided
+						'label_before' => '<dt class="tribe-events-event-categories-label">',
+						'label_after'  => '</dt>',
+						'wrap_before'  => '<dd class="tribe-events-event-categories">',
+						'wrap_after'   => '</dd>',
+				]
```
Comment (paulmskim) [inline src/views/modules/meta/details.php:150]: indentation

File: src/views/blocks/parts/details.php [hunk 7]
```diff
@@ -147,16 +147,17 @@
...
+				[
+						'before'       => '',
+						'sep'          => ', ',
+						'after'        => '',
+						'label'        => null, // An appropriate plural/singular label will be provided
+						'label_before' => '<dt class="tribe-events-event-categories-label">',
+						'label_after'  => '</dt>',
+						'wrap_before'  => '<dd class="tribe-events-event-categories">',
+						'wrap_after'   => '</dd>',
+				]
```
Comment (paulmskim) [inline src/views/blocks/parts/details.php:160]: indentation

File: src/admin-views/aggregator/fields/schedule.php [hunk 8]
```diff
@@ -1,23 +1,23 @@
...
-	);
+	$month_days[] = [
+			'id'   => $i,
+			'text' => (string) $i,
+	];
 }
 ?>
 <tr
-	class="tribe-dependent"
+		class="tribe-dependent"
```
Comment (paulmskim) [inline src/admin-views/aggregator/fields/schedule.php:20]: indentation

File: src/admin-views/aggregator/origins/eventbrite.php [hunk 9]
```diff
@@ -2,14 +2,17 @@
...
 $origin_slug        = 'eventbrite';
-$field              = (object) array();
+$field              = (object) [];
 $field->label       = __( 'Import Type:', 'the-events-calendar' );
 $field->placeholder = __( 'Select Import Type', 'the-events-calendar' );
-$field->help        = __( 'One-time imports include all currently listed events, while scheduled imports automatically grab new events and updates from Eventbrite on a set schedule. Single events can be added via a one-time import.', 'the-events-calendar' );
+$field->help        = __(
+		'One-time imports include all currently listed events, while scheduled imports automatically grab new events and updates from Eventbrite on a set schedule. Single events can be added via a one-time import.',
+		'the-events-calendar'
+);
```
Comment (paulmskim) [inline src/admin-views/aggregator/origins/eventbrite.php:11]: this and this file has weird indentation

File: src/admin-views/aggregator/origins/gcal.php [hunk 10]
```diff
@@ -1,15 +1,21 @@
...
 $origin_slug        = 'gcal';
-$field              = (object) array();
+$field              = (object) [];
 $field->label       = __( 'Import Type:', 'the-events-calendar' );
 $field->placeholder = __( 'Select Import Type', 'the-events-calendar' );
-$field->help        = __( 'One-time imports include all events in the current feed, while scheduled imports automatically grab new events and updates from the feed on a set schedule.', 'the-events-calendar' );
+$field->help        = __(
+		'One-time imports include all events in the current feed, while scheduled imports automatically grab new events and updates from the feed on a set schedule.',
+		'the-events-calendar'
+);
```
Comment (paulmskim) [inline src/admin-views/aggregator/origins/gcal.php:10]: this and this file has weird indentation

File: src/admin-views/aggregator/origins/ical.php [hunk 11]
```diff
@@ -1,15 +1,21 @@
...
 $origin_slug        = 'ical';
-$field              = (object) array();
+$field              = (object) [];
 $field->label       = __( 'Import Type:', 'the-events-calendar' );
 $field->placeholder = __( 'Select Import Type', 'the-events-calendar' );
-$field->help        = __( 'One-time imports include all events in the current feed, while scheduled imports automatically grab new events and updates from the feed on a set schedule.', 'the-events-calendar' );
+$field->help        = __(
+		'One-time imports include all events in the current feed, while scheduled imports automatically grab new events and updates from the feed on a set schedule.',
+		'the-events-calendar'
+);
```
Comment (paulmskim) [inline src/admin-views/aggregator/origins/ical.php:10]: this and this file has weird indentation

File: src/admin-views/aggregator/origins/ics.php [hunk 12]
```diff
@@ -39,7 +39,14 @@ class="tribe-ea-field tribe-ea-media_button tribe-dependent button button-second
...
 if ( 'edit' === $tab->get_slug() ) {
-	$this->template( 'fields/schedule', array( 'record' => $record, 'origin' => $origin_slug, 'aggregator_action' => $aggregator_action ) );
+	$this->template(
+			'fields/schedule',
+			[
+					'record'            => $record,
+					'origin'            => $origin_slug,
+					'aggregator_action' => $aggregator_action,
+			]
+	);
```
Comment (paulmskim) [inline src/admin-views/aggregator/origins/ics.php:49]: indentation

File: src/admin-views/aggregator/origins/meetup.php [hunk 13]
```diff
@@ -1,15 +1,21 @@
...
 $origin_slug        = 'meetup';
-$field              = (object) array();
+$field              = (object) [];
 $field->label       = __( 'Import Type:', 'the-events-calendar' );
 $field->placeholder = __( 'Select Import Type', 'the-events-calendar' );
-$field->help        = __( 'One-time imports include all currently listed events, while scheduled imports automatically grab new events and updates from Meetup on a set schedule. Single events can be added via a one-time import.', 'the-events-calendar' );
+$field->help        = __(
+		'One-time imports include all currently listed events, while scheduled imports automatically grab new events and updates from Meetup on a set schedule. Single events can be added via a one-time import.',
+		'the-events-calendar'
+);
```
Comment (paulmskim) [inline src/admin-views/aggregator/origins/meetup.php:10]: this and this file has weird indentation

File: src/admin-views/aggregator/origins/url.php [hunk 14]
```diff
@@ -72,12 +72,19 @@ class="tribe-bumpdown-trigger tribe-bumpdown-permanent tribe-bumpdown-nohover tr
...
 if ( 'edit' === $tab->get_slug() ) {
-	$this->template( 'fields/schedule', array( 'record' => $record, 'origin' => $origin_slug, 'aggregator_action' => $aggregator_action ) );
+	$this->template(
+			'fields/schedule',
+			[
+					'record'            => $record,
+					'origin'            => $origin_slug,
+					'aggregator_action' => $aggregator_action,
+			]
+	);
```
Comment (paulmskim) [inline src/admin-views/aggregator/origins/url.php:82]: indentation

File: src/admin-views/aggregator/settings.php [hunk 15]
```diff
@@ -19,12 +19,15 @@
...
 
-$origin_categories = array(
-	'-1' => $events_aggregator_is_active ? $use_global_settings_phrase : esc_html__( 'None', 'the-events-calendar' ),
-);
+$origin_categories = [
+		'-1' => $events_aggregator_is_active ? $use_global_settings_phrase : esc_html__(
+				'None',
+				'the-events-calendar'
+		),
+];
```
Comment (paulmskim) [inline src/admin-views/aggregator/settings.php:30]: this and this file has weird indentation

File: src/admin-views/aggregator/status.php [hunk 16]
```diff
@@ -33,7 +33,9 @@
 				$notes .= '</a>';
 			} else {
 				$text  = __( 'Your license is invalid', 'the-events-calendar' );
-				$notes = '<a href="' . esc_url( Tribe__Settings::instance()->get_url( array( 'tab' => 'licenses' ) ) ) . '">' . esc_html__( 'Check your license key', 'the-events-calendar' ) . '</a>';
+				$notes = '<a href="' . esc_url(
+								Tribe__Settings::instance()->get_url( [ 'tab' => 'licenses' ] )
+						) . '">' . esc_html__( 'Check your license key', 'the-events-calendar' ) . '</a>';
```
Comment (paulmskim) [inline src/admin-views/aggregator/status.php:38]: this and this file has weird indentation

File: src/admin-views/tribe-options-display.php [hunk 17]
```diff
@@ -90,94 +90,94 @@
...
-		'info-end'                           => array(
+			          . __( 'The settings below control the display of your calendar. If things don\'t look right, try switching between the three style sheet options or pick a page template from your theme.', 'the-events-calendar' )
+			          . '</p> <p>'
+			          . sprintf(
+			          /* translators: %s: URL to knowledgebase. Please also use &#37; for % to avoid PHP warnings. */
+				          __( 'There are going to be situations where no out-of-the-box template is 100&#37; perfect. Check out our <a href="%s">our themer\'s guide</a> for instructions on custom modifications.', 'the-events-calendar' ),
+				          Tribe__Main::$tec_url . 'knowledgebase/themers-guide/?utm_medium=plugin-tec&utm_source=generaltab&utm_campaign=in-app'
+			          )
+			          . '</p>',
+		],
```
Comment (paulmskim) [inline src/admin-views/tribe-options-display.php:113]: indentation? also uses spaces.

File: src/deprecated/Tribe__Events__Importer__Options.php [hunk 18]
```diff
@@ -5,10 +5,10 @@
...
  */
 class Tribe__Events__Importer__Options {
-	public static $options = array(
-		'imported_post_status',
-		'imported_encoding_status',
-	);
+	public static $options = [
+			'imported_post_status',
+			'imported_encoding_status',
+	];
```
Comment (paulmskim) [inline src/deprecated/Tribe__Events__Importer__Options.php:11]: this and this file has weird indentation

File: src/deprecated/facebook.php [hunk 19]
```diff
@@ -103,12 +103,15 @@ class="tribe-bumpdown-trigger tribe-bumpdown-permanent tribe-bumpdown-nohover tr
 
 <?php
 if ( 'edit' === $tab->get_slug() ) {
-	$this->template( 'fields/schedule', array( 'record' => $record, 'origin' => $origin_slug, 'aggregator_action' => $aggregator_action ) );
+	$this->template( 'fields/schedule', [ 'record'            => $record,
+										  'origin'            => $origin_slug,
+										  'aggregator_action' => $aggregator_action,
+	] );
```
Comment (paulmskim) [inline src/deprecated/facebook.php:109]: indentation

File: src/functions/php-min-version.php [hunk 20]
```diff
@@ -91,10 +91,10 @@ function tribe_not_php_version_names() {
 	 *
 	 * @param array $names Name of the plugins that are not compatible
 	 */
-	return apply_filters( 'tribe_not_php_version_names', array() );
+	return apply_filters( 'tribe_not_php_version_names', [] );
 }
 
-/**
+	/**
```
Comment (paulmskim) [inline src/functions/php-min-version.php:97]: indentation?

File: src/views/blocks/event-category.php [hunk 21]
```diff
@@ -15,18 +15,18 @@
...
+					'before'       => '',
+					'sep'          => ', ',
+					'after'        => '',
+					'label'        => null, // An appropriate plural/singular label will be provided
+					'label_before' => '<dt class="tribe-events-event-categories-label">',
+					'label_after'  => '</dt>',
+					'wrap_before'  => '<dd class="tribe-events-event-categories">',
+					'wrap_after'   => '</dd>',
+			]
+	);
```
Comment (paulmskim) [inline src/views/blocks/event-category.php:30]: indentation

**Review by paulmskim — COMMENTED** [review]
The code looks good, but there are a number of files that have extra indentation or indentation with spaces. Maybe an editor setting?

**Comment by mitogh** [comment]
Thank you for the review @lucatume @paulmskim

**Review by lucatume — DISMISSED** [review]
Solve the conflicts, then good to merge.

**Comment by lucatume** [comment]
Rebasing on `release/B20.12`.

**Pull request merged**